RiotSharpTest/RiotApiTest.cs

```diff
@@ -137,20 +137,24 @@ public void GetChampionAsync_Test()
         [TestCategory("RiotApi")]
         public void GetMasteryPages_Test()
         {
-            var masteries = api.GetMasteryPages(RiotApiTestBase.summonersRegion, RiotApiTestBase.summonerIds);
+            var fetchCount = Math.Min(10, RiotApiTestBase.summonerIds.Count());
```

Comment:
Since batching is going away, we can limit ourselves to keeping only 10 summoner ids, what do you think?